asgard/tests/benchmark_projector_cache.cpp

```diff
@@ -534,8 +536,14 @@ ListOfLocations build_list_of_locations(size_t nb_threads) {
                                           "coord:2.292557:48.825697",
                                           "coord:2.284232:48.793203",
                                           "coord:2.304749:48.811078"};
+
+    std::vector<valhalla::midgard::PointLL> pointLLs;
+    std::transform(locations.begin(), locations.end(), std::back_inserter(pointLLs), [](const auto& coord) {
+        return valhalla::midgard::PointLL{navitia::parse_coordinate(coord)};
+    });
```

Comment:
If you have boost with Asgard (I don't think you do :thinking: ). You could save up some writting :)
```suggestion
    boost::range::transform(locations, std::back_inserter(pointLLs), [](const auto& coord) {
        return valhalla::midgard::PointLL{navitia::parse_coordinate(coord)};
    });
```